system/docker/node/Dockerfile

```diff
@@ -47,10 +47,11 @@ ARG SOVRIN_VERSION
 ARG SOVTOKEN_VERSION
 ARG SOVTOKENFEES_VERSION
 ENV TOKEN_PLUGINS_INSTALL=$TOKEN_PLUGINS_INSTALL
-ENV SOVRIN_VERSION=${SOVRIN_VERSION:-1.1.46}
-ENV SOVTOKEN_VERSION=${SOVTOKEN_VERSION:-0.9.13}
+ENV SOVRIN_VERSION=${SOVRIN_VERSION:-1.1.50}
+ENV SOVTOKEN_VERSION=${SOVTOKEN_VERSION:-0.9.13~dev58}
```

Comment:
Why dev version? Also it looks rather old, recent ones are 1.0.0

Reply:
It's just default version that will be increased as confirmation testing will go ahead and it doesn't affect CD pipeline since all packages' versions are declared explicitly there.